src/main/java/com/minecolonies/coremod/blocks/BlockMinecoloniesRack.java

```diff
@@ -192,7 +192,17 @@ public int getMetaFromState(@NotNull final IBlockState state)
         return state.getValue(FACING).getHorizontalIndex();
     }
 
+    /**
+     * Does some action if a neighbor changes.
+     * @deprecated but we definitely need it.
+     * @param state the state.
+     * @param worldIn the world.
+     * @param pos the pos.
+     * @param blockIn the block.
+     * @param fromPos the changed ps.
+     */
     @Override
+    @Deprecated
     public void neighborChanged(final IBlockState state, final World worldIn, final BlockPos pos, final Block blockIn, final BlockPos fromPos)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1133)